frontend/src/components/Dialog/Dialog.vue

```diff
@@ -35,29 +35,14 @@
     </vue-final-modal>
 </template>
 
-<script>
-export default {
-    inheritAttrs: false,
-    emits: ['confirm'],
-    props: {
-        message: {
-            type: String,
-            default: '',
-        },
-        title: {
-            type: String,
-            default: '',
-        },
-        confirmText: {
-            type: String,
-            default: 'Confirm',
-        },
-        cancelText: {
-            type: String,
-            default: 'Cancel',
-        },
-    },
-};
+<script setup lang="ts">
+const emit = defineEmits(['confirm']);
```

Comment:
Hi Nick, sorry I checked the documentation again, seems like we don't need to do the emit declaration, if we just use $emit in the html template. Sorry about that, think we could remove this. https://vuejs.org/guide/components/events.html#event-arguments

Reply:
No problem, I have reverted back this change.